### ISSUE TITLE
libaec wrong md5sum

### ISSUE THREAD

**Issue by gustavoberman** [spack/spack#7922]
If you want to report a build error for some package, or a bug in core please use the following template as a guideline:

-----

Summary

### Expected Result

libaec compile!

### Actual Result

md5 is wrong so it cannot compile
md5 for libaec 1.0.2 should be: ef388abe8e6d6928e54f41bc63e12de4

### Steps to reproduce the issue

```console
$ spack install hdf5 +cxx +fortran +szip

...
```

**Comment by adamjstewart** [comment]
I believe this should be fixed by #7881 (just merged). Can you update to the latest develop commit and try again? You may need to purge the download cache (either manually delete the tarball that was downloaded or run `spack clean --downloads`).

**Comment by alalazo** [comment]
This seems to work at 5eca4f14706a9590407f14c9c76e5a8bb663e9df:
```console
$ spack fetch libaec
[ ... ]
==> Fetching https://gitlab.dkrz.de/api/v4/projects/k202009%2Flibaec/repository/archive.tar.gz?sha=v1.0.2
```
Closing the issue.
